exonum/benches/criterion/block.rs

```diff
@@ -106,7 +106,12 @@ fn create_blockchain(db: impl Database, services: Vec<Box<dyn Service>>) -> Bloc
 }
 
 fn execute_block(blockchain: &Blockchain, height: u64, txs: &[Hash]) -> (Hash, Patch) {
-    blockchain.create_patch(ValidatorId::zero(), Height(height), txs)
+    blockchain.create_patch(
+        ValidatorId::zero(),
+        Height(height),
+        txs,
+        &mut BTreeMap::new(),
```

Comment:
```suggestion
        None,
```
I suppose `BTreeMap::new` not free operation.

Reply:
https://github.com/rust-lang/rust/blob/master/src/liballoc/collections/btree/node.rs#L127
It seems that `BtreeMap::new` is cheap operation.

Reply:
But benchmarks say otherwise :)

Reply:
This code with `BtreeMap::new` is used only in benchmarks or tests. It doesn't affect the production code. But adding `Option` will result in unnecessary unwraps.